editor/src/clj/editor/editor_extensions/ui_components.clj

```diff
@@ -403,19 +391,13 @@
                             (fx.mutator/property-change-listener #(.selectedProperty ^CheckBox %))
                             property-change-listener-with-source-owner-window-lifecycle)}))
 
-(defn- apply-tooltip-and-issue [props tooltip issue localization-state]
-  (let [message (:message issue)]
-    (cond-> props (or message tooltip) (apply-tooltip (if (and message tooltip)
-                                                        (localization/join "\n\n" [message tooltip])
-                                                        (or message tooltip))
-                                                      localization-state))))
-
 (defn- set-tooltip-and-issue [props tooltip issue localization-state]
   (let [message (:message issue)]
-    (cond-> props (or message tooltip) (assoc :tooltip (localization-state
-                                                         (if (and message tooltip)
-                                                           (localization/join "\n\n" [message tooltip])
-                                                           (or message tooltip)))))))
+    (cond-> props (or message tooltip) (assoc :tooltip {:severity (:severity issue :info)
+                                                        :message (localization-state
+                                                                   (if (and message tooltip)
+                                                                     (localization/join "\n\n" [message tooltip])
+                                                                     (or message tooltip)))}))))
```

Comment:
We could take this opportunity to split this expression into multiple lines to fight the rather severe indentation here.